README.adoc

```diff
@@ -124,15 +124,34 @@ can be done immediately; no need to wait for confirmations) by calling
 
 `Bridge.revealDeposit` takes in the funding transaction, and then the necessary
 information to reconstruct the Bitcoin script: `depositor`, `blindingFactor`,
-`walletPubKeyHash` and `refundPubkeyHash`. Then, it makes sure that the funding
-transaction happened on bitcoin, reconstructs the script, hashes it, verifies
-that the hashes match, and then stores the deposit as waiting to be
-<sweeping,swept> associated to the provided `depositor`.
+`walletPubKeyHash` and `refundPubkeyHash`. Then it reconstructs the script,
+hashes it, verifies that the hashes match, and then stores the deposit as
+waiting to be <sweeping,swept> associated to the provided `depositor`.
 
 [sweeping]
 ==== Sweeping
 
+Periodically, off-chain clients associated to a wallet collect a batch of
+deposits and create a sweep transaction. This transaction includes all of the
+deposit UTXOs as well as the wallet's UTXO (`Wallet.mainUtxoHash`) as inputs
```

Comment:
Maybe: "This transaction includes revealed and valid deposit UTXOs" ?

I see two reasons:
1. Wallet is not including a revealed deposit if something is wrong with it (for example, lock time). That's why I think *all* could be confusing.
2. There is no guarantee wallet will include all valid revealed deposits since the last sweep given the problems with proving too large transactions on Ethereum. Wallet may need to process them in batches.

Reply:
[0088980](https://github.com/keep-network/tbtc-v2/pull/332/commits/008898070030619e662700b93d1fd8de90b059e3)

woah difficulty of 2